src/components/organisms/news/news.scss

```
@@ -0,0 +1,48 @@
+@use "../../variables" as *;
+.news {
+  color: $black;
+  .news-and-events-title {
+    & > h2 {
+      font-size: $fontsize-xxl;
+    }
+  }
+  .frame1 {
+    background-color: $white;
+    padding-bottom: 30px;
+    .wrapper {
+      padding: 20px;
+      display: flex;
+      flex-direction: column;
+      gap: 10px;
+      border: solid $gray_gainsboro;
+      border-width: 1px; 
+      
+
+      + .wrapper {
+        border-top: none;
+      }
+
+      time {
+        color: $gray_nobel;
+        font-size: $fontsize-xs;
+        line-height: 12px;

```

```suggestion
        line-height: 1;
```
font-size と同じにしたい時はこのようにかけます。
line-heightは単位なしが推奨されているので、可能な時はこれを使っていきましょう「。

> <number> (単位なし)
使用値は、この単位のない [<number>](https://developer.mozilla.org/ja/docs/Web/CSS/number) に要素のフォントサイズを掛けたものになります。計算値は、指定された <number> と同じです。ほとんどの場合、継承時の予期しない結果を避けるために、これが line-height を設定する好ましい方法です


https://developer.mozilla.org/ja/docs/Web/CSS/line-height